src/particles/diagnostics/ReducedBeamCharacteristics.cpp

```diff
@@ -322,6 +400,9 @@ namespace impactx::diagnostics
         data["dispersion_y"] = dispersion_y;
         data["dispersion_py"] = dispersion_py;
         data["charge_C"] = charge;
+        data["emittance_1"] = emittance_1;
```

Comment:
As discussed in the developer meeting last Wed:
Instead of redefining the meaning of `emittance_N`, we rather would skip writing it if the diagnostics was disabled.